os/arch/arm/src/imxrt/imxrt_lowputc.c

```diff
@@ -545,4 +555,64 @@ void imxrt_lowputc(int ch)
 	while ((getreg32(IMXRT_CONSOLE_BASE + IMXRT_LPUART_STAT_OFFSET) & LPUART_STAT_TDRE(1U)) == 0) {
 	}
 }
+
+/****************************************************************************
+ * Name: imxrt_disable_rxint
+ *
+ * Description:
+ *   Call to enable or disable RX interrupts
+ *
+ ****************************************************************************/
+void imxrt_set_rxint(bool enable)
+{
+	uint32_t regval;
+	uint32_t mask;
+	uint32_t rdtemp;
+
+	if (enable) {
+		mask = (LPUART_CTRL_RIE(1U) | LPUART_CTRL_FEIE(1U) | LPUART_CTRL_ORIE(1U));
+		rdtemp = getreg32(IMXRT_CONSOLE_BASE + IMXRT_LPUART_CTRL_OFFSET);
+		rdtemp |= mask;
+		putreg32(rdtemp, IMXRT_CONSOLE_BASE + IMXRT_LPUART_CTRL_OFFSET);
+	} else {
+		mask = ~(LPUART_CTRL_RIE(1U) | LPUART_CTRL_FEIE(1U) | LPUART_CTRL_ORIE(1U));
+		rdtemp = getreg32(IMXRT_CONSOLE_BASE + IMXRT_LPUART_CTRL_OFFSET);
+		rdtemp &= mask;
+		putreg32(rdtemp, IMXRT_CONSOLE_BASE + IMXRT_LPUART_CTRL_OFFSET);
+	}
+}
+
+/****************************************************************************
+ * Name: up_getc
+ *
+ * Description:
+ *   Get one character from the UART
+ *
+ ****************************************************************************/
+uint8_t up_getc(void)
+{
+	imxrt_set_rxint(0);
```

Comment:
up_getc could be called from normal operation. Then, enabling rxint is necessary.

Reply:
yes, for future use case, I have added enabling rxint.